{{ cookiecutter.hosting_github_repo_name }}/web/src/index.jsx

```diff
@@ -0,0 +1,27 @@
+{% raw %}
+import React from "react";
+import ReactDOM from "react-dom";
+import { App, AppConfigProvider } from "@councildataproject/cdp-frontend";
+
+ReactDOM.render(
+    <div>
+        <AppConfigProvider
+            appConfig={{
+                firebaseConfig: {
+                    options: {
+                        projectId: {% endraw %}"{{ cookiecutter.infrastructure_slug }}"{% raw %},
+                    },
+                    settings: {},
+                },
+                municipality: {
+                    name: {% endraw %}"{{ cookiecutter.municipality }}"{% raw %},
+                    footerLinksSections: [],
+                },
+            }}
+        >
```

Comment:
Since this object is resulting in some funky code formatting/line wraps of the AppConfigProvider component, could we pull this object out as a variable and place it above the markup?

For example:
```
const config = { ... };

...other code...

<AppConfigProvider appConfig={config}>
    <App />
</AppConfigProvider>
```

Goal is just to make things a little more readable. 

However, since this my personal stylistic preference you're welcome to ignore or argue against this idea.

Reply:
nah I agree. Will do! If you got the time, would love your :eyes: on the matching cdp-frontend PR because oooooooh boy webpack + snowpack is a doozy for me.

Reply:
Whoops missed this. But oh yeahhh webpack + snowpack is a still doozy for me most of the time.

Reply:
Happy to take a peek at that PR assuming it hasn't already been merged.

Reply:
It's been merged but still broken hahaha

Reply:
Hahaha oh noo 😂

Reply:
I can potentially try to do a light pass on it and see if anything stands out

Reply:
I am a bit busy with other stuff this week. The most recent commits I pushed to this branch make it possible to render the app at the least. (It's not a good render)

I think the thing is that the cdp-frontend library isn't packaging up any of the styles we need.

If you want to push directly to this branch to try stuff feel free. (although without releasing new versions of cdp-frontend i am not sure how much we can iteratively test)